### PR TITLE
Edit publisher 

### LINKED ISSUE #51: Edit publisher.js
Import all events directly to publisher.js

### PR DESCRIPTION
**Pull request by MozdolevskyiMaksym**
Publisher is edited. Import all events to publisher.

### CODE REVIEW

File: app/helpers/publisher.js [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
 
-  unsubscribe = (event, callbackFn) => {
-    if (!this.subscribers[event]) {
-      this.subscribers[event] = [];
+  unsubscribe = (eventName, callbackFn) => {
+    if(!this.events[eventName]) {
+      this.events[eventName] = [];
     }
-    this.subscribers[event] = this.subscribers[event].filter((func) => func !== callbackFn);
+    this.events[eventName] = this.events[eventName].filter(func => func !== callbackFn)
```
Comment (PavelKundenko) [inline app/helpers/publisher.js]: Our eslint requires using parentheses around arrow function argument
`filter((func) => func !== callbackFn)`

File: app/helpers/publisher.js [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
 
-  notify = (event, data) => {
-    if (!this.subscribers[event]) {
-      this.subscribers[event] = [];
+  notify = (eventName, data) => {
+    if(!this.events[eventName]){
+      this.events[eventName] = [];
     }
-    this.subscribers[event].forEach((func) => func(data));
+    this.events[eventName].forEach(func => func(data));
```
Comment (PavelKundenko) [inline app/helpers/publisher.js]: Same as previous

File: app/helpers/publisher.js [hunk 3]
```diff
@@ -1,32 +1,34 @@
...
-    subscribe: this.subscribe,
-    unsubscribe: this.unsubscribe,
-    notify: this.notify,
-  });
-}
+  methods = {
+    notify      : this.notify,
+    subscribe   : this.subscribe,
+    events      : this.eventNames
+  }
```
Comment (PavelKundenko) [inline app/helpers/publisher.js:33]: 1) Add unsubscribe to methods obj. 
2) Our eslint doesn't like extra spaces. I think it's better to remove them.

**Review by PavelKundenko — APPROVED** [review]
Good

**Review by rearchontis — APPROVED** [review]

**Review by iklopov — APPROVED** [review]

**Review by Winchestery — COMMENTED** [review]
>

**Pull request merged**